validation/mobile-service-broker/pom.xml

```diff
@@ -17,6 +17,7 @@
     <name>Java integration for SAP Mobile Services - Mobile Services Broker for Validation</name>
 
     <properties>
+        <java.version>11</java.version>
```

Comment:
Works for me. However, afaik the buildpack already supports Java 17.
Otherwise we can also migrate to the community buildpack, which definitely supports Java 17.

Once I have time, I'll investigate this => would then resolve it in a separate PR.

Reply:
I had to learn, that the current Java Build Pack still doesn't support Spring Framework 6, Spring Boot 3 and Java 17 - unofficial timeline is Q3/Q4.